src/jobs/send-mail.ts

```diff
@@ -12,42 +12,43 @@ const viewsDir = path.join(__dirname, "../emails");
 
 const sendMail = async (job: any) => {
   try {
-    if (process.env.SENDGRID_API_KEY) {
-      const email = new Email({
-        message: {
-          from: `"Mirlo" <${
-            process.env.SENDGRID_SENDER ?? "no-reply@mirlo.space"
-          }>`,
+    const email = new Email({
+      message: {
+        from: `"Mirlo" <${
+          process.env.SENDGRID_SENDER ?? "no-reply@mirlo.space"
+        }>`,
+      },
+      juice: true,
+      send: true,
+      juiceResources: {
+        preserveImportant: true,
+        webResources: {
+          relativeTo: path.resolve(viewsDir),
         },
-        juice: true,
-        send: true,
-        juiceResources: {
-          preserveImportant: true,
-          webResources: {
-            relativeTo: path.resolve(viewsDir),
-          },
-        },
-        transport: nodemailer.createTransport(
-          sendgrid({
-            apiKey: process.env.SENDGRID_API_KEY,
-          })
-        ),
+      },
+      transport: !!process.env.SENDGRID_API_KEY
```

Comment:
Instead of requiring the API key up front, check it when setting the transport. Note that with this control flow `Email.send` can still be called in a development environment, although currently it's only called in production.

Reply:
So should we remove the production check before `email.send` below and just have it be a json output instead of logging the rest of the e-mail?

Reply:
Can you clarify with a proposed code snippet? I'm not sure of your proposal. My goal with this PR was to see emails come through the log so I could confirm new accounts in a dev env, without having the Sendgrid API key. It's possible there's a better solution that doesn't involve code changes, but feels valuable for dev env to work without external services

Reply:
For sure, I think I just oversaw that scenario because I had a string in the .env file. 

I'll comment where I think the change could happen, but it's also fine without it.

Reply:
I'll have to test that it does log with `.send` (unless you've tested it)

Reply:
(I haven't!)